src/module/canvas/status-effects.ts

```diff
@@ -281,16 +280,14 @@ export class StatusEffects {
             if (objectHasKey(CONFIG.PF2E.conditionTypes, slug)) {
                 const newCondition = game.pf2e.ConditionManager.getCondition(slug).toObject();
                 await token.actor?.createEmbeddedDocuments("Item", [newCondition]);
-            } else if (iconSrc && (event.shiftKey || control.dataset.statusId === "dead")) {
```

Comment:
I wonder what shift key did here, I'm testing in live and its not doing anything. I'll assume what you did is correct.

Reply:
I think it was supposed to set the icon as overlay effect. I'm not sure when that broke but from the lack of an issue I'm assuming no one used it.

Reply:
I tested in 5e and I can't get the icon to overlay anyways via the condition hud.